C2CRIBuildDir/projects/C2C-RI/src/RIGUI/src/org/fhwa/c2cri/testmodel/TestConfiguration.java

```diff
@@ -108,7 +107,7 @@ public TestConfiguration() {
      * @param apps the apps
      * @param info the info
      */
-    public TestConfiguration(TestMode modes, TestOptions options, SUT sut, RILoggingInfo log, DefaultLayerParameters apps, DefaultLayerParameters info) {
```

Comment:
This log parameter is also in the Javadoc of this constructor.

Reply:
Removed the parameter from the Javadoc